### ISSUE TITLE
hi ollie its not getting non eu streams idk why

### ISSUE THREAD

**Issue by nmro97** [O11iee/TTAutoRecord#8]
found room id comes through but it dosnt download anyhting , it hat a restriction earlier maybe its cuz of thath ?

**Comment by nmro97** [comment]
hi so its not getting some of the streams, after the restriction can u check

**Comment by O11iee** [comment]
Seems they have added an expiry on the pull URLs, i will have to adjust the regex.

**Comment by nmro97** [comment]
ok thank you m8

**Comment by O11iee** [comment]
https://github.com/O11iee/TTAutoRecord/releases/tag/v4.0.1-alpha

**Comment by nmro97** [comment]
is it me or is it restricted again , its says adjust your settings rn

**Comment by nmro97** [comment]
yep still not getting non eu ones i think titkok just restricted or something

**Comment by nmro97** [comment]
the 4.00 version works with usa  vpn

**Comment by nmro97** [comment]
![tt](https://github.com/O11iee/TTAutoRecord/assets/149392722/78b2b22a-74ad-4568-a4aa-2ae40e07ff22)
hi ollie   now its not working at all, all i get is that , tiktok dit something

**Comment by shadowdragon245** [comment]
maybe try and update your cookies

**Comment by nmro97** [comment]
TY but tiktok is restricting yea

**Comment by nmro97** [comment]
hi ollie i wanted to give an update, so im using the older one however it wont get the non eu streams as i said, but the newer one u dit keeps restarting the download and it takes always couple min to get the stream back  but it gets almost every streams . hope this helps
